script.js

```diff
@@ -79,42 +79,61 @@ function shuffleCards() {
 
 function compareCards(e) {
   let clickedCard = e.target;
-  clickedCard.classList.add("flipped");
+  selectCard(clickedCard);
 
-  let flippedCards = document.querySelectorAll(".flipped");
+  let selectedCards = document.querySelectorAll(".selected");
 
   //Logic
-  if (flippedCards.length === 2) {
-    if (
-      flippedCards[0].getAttribute("name") ===
-      flippedCards[1].getAttribute("name")
-    ) {
-      console.log("match");
-      winCondition();
-      flippedCards[0].style.pointerEvents = "none";
-      flippedCards[1].style.pointerEvents = "none";
-    } else {
-      console.log("wrong");
-      setTimeout(function () {
-        flippedCards[0].classList.remove("toggleCard");
-        flippedCards[1].classList.remove("toggleCard");
-      }, 1000);
-    }
-    flippedCards[0].classList.remove("flipped");
-    flippedCards[1].classList.remove("flipped");
+  if (selectedCards.length !== 2) {
+    return;
```

Comment:
To leave the function when the condition is not met. It helps avoid nested ifs

Reply:
Code does nothing, function will automatically end as following if statement will run to end of function